### PR TITLE
Feat/client implementation of datamodel server

### PR DESCRIPTION
**Pull request by prmukherj**
Hi, please have a look at this implementation and let me know your feedback. Thank you.

### CODE REVIEW

**Review by seanpearsonuk — APPROVED** [review]

File: src/ansys/fluent/core/_stand_alone_datamodel_client/_preferences_client.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+"""Sample testing code (next)."""
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/_stand_alone_datamodel_client/_preferences_client.py:1]: @prmukherj just wondering if we need all this code

**Review by hpohekar — APPROVED** [review]

File: src/ansys/fluent/core/_stand_alone_datamodel_client/_preferences_client.py (hunk 1 above)
Comment (prmukherj) [reply src/ansys/fluent/core/_stand_alone_datamodel_client/_preferences_client.py:1]: @seanpearsonuk, yes something like that can be done. I just checked it and it seems to be working. I'll update this code as required. Thank you.

Comment (prmukherj) [reply src/ansys/fluent/core/_stand_alone_datamodel_client/_preferences_client.py:1]: @seanpearsonuk and @mkundu1, does the current implementation look good?

Comment (seanpearsonuk) [reply src/ansys/fluent/core/_stand_alone_datamodel_client/_preferences_client.py]: @prmukherj please go ahead and merge. Thanks.

Comment (seanpearsonuk) [reply src/ansys/fluent/core/_stand_alone_datamodel_client/_preferences_client.py]: Next set of changes after merge, based on this kind of thing:
```
>>> from ansys.fluent.core.services.datamodel_se import DatamodelService, PyMenuGeneric
>>> pref_root = PyMenuGeneric(s.fluent_connection.datamodel_service_se, "preferences")
>>> pref_root.MeshingWorkflow()
{'CheckpointingOption': 'Write mesh files', 'DrawSettings': {'FaceZoneLimit': 10000, 'AutoDraw': 'auto', 'FacetLimit': 5000000}, 'SaveCheckpointFiles': True, 'TempFolder': '', 'TemplatesFolder': '', 'Verbosity': 'off'}
>>>
```

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**